src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Color {
+    GREEN, BLUE, RED, BLACK;
```

Comment:
it's clearer to define enums line by line, and not all in one line

Reply:
not fixed